src/index.d.ts

```diff
@@ -21,8 +21,8 @@ export declare function shouldForwardProp(
   predicate: (x: string) => boolean
 ): (props: string[]) => string[];
 export declare function setup(
-  prefixer: null | ((key: string, value: any) => string),
-  shouldForwardProp?: null | ((props: string[]) => string[])
+  prefixer?: (key: string, value: any) => string,
```

Comment:
Looking at the `goober` TS provided, we technically should not be forwarding `null` into `gooberSetup` since it does not specify `null` as a valid parameter type. Rather, it should be `undefined`. However, if this is a change we do not want to make, I understand.

Reply:
Seems fine. this was added after the fact in PR. If it works I have no issue. I just don't 100% trust goobers types.